entity-framework/core/what-is-new/ef-core-8.0/plan.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Plan for Entity Framework Core 8
+description: The themes and features planned for EF Core 8
+author: ajcvickers
+ms.date: 12/13/2022
+uid: core/what-is-new/ef-core-8.0/plan
+---
+
+# Plan for Entity Framework Core 8
+
+![EF8 Logo](ef8.png)
+
+As described in the [planning process](xref:core/what-is-new/release-planning), we have gathered input from stakeholders into a plan for Entity Framework Core 8 (EF Core 8) and other .NET Data Access work for the .NET 8 timeframe. For brevity, EF Core 8 is also referred to as just EF8.
+
+> [!IMPORTANT]
+> This plan is not a commitment; it will evolve as we continue to learn throughout the release. Some things not currently planned for EF8 may get pulled in. Some things currently planned for EF8 may get punted out.
+
+## General information
+
+EF Core 8 is the next release after EF Core 7 and is scheduled for release in November 2023, at the same time as .NET 8. There are no plans for an EF Core 7.1 release.
+
+### Supported platforms
+
+EF8 currently targets .NET 6. This will likely be updated to .NET 8 as we near the release. EF8 does not target any .NET Standard version; for more information see [the future of .NET Standard](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/). EF8 will not run on .NET Framework.
```

Comment:
No .NET Framework - is that relevant any more?

Reply:
Better to explicitly state it just so it's 100% clear.